buildSrc/src/main/kotlin/com/datadog/gradle/config/AndroidConfig.kt

```diff
@@ -20,7 +20,7 @@ object AndroidConfig {
     const val MIN_SDK_FOR_WEAR = 23
     const val BUILD_TOOLS_VERSION = "35.0.0"
 
-    val VERSION = Version(2, 15, 0, Version.Type.Snapshot)
+    val VERSION = Version(2, 15, 0, Version.Type.Release)
```

Comment:
I merged https://github.com/DataDog/dd-sdk-android/pull/2346, so please sync with `develop` again - this will create a conflict which should be resolved by ignoring the change.

Reply:
updated